components/molecules/medium-title-intro.tsx

```diff
@@ -1,15 +1,21 @@
+import {classNames} from 'notion-to-tailwind'
+
 type MediumTitleIntroProps = {
   title: string
+  subTitle?: string
   textColLeft?: string
   textColRight?: string
+  className?: string
 }
 
-export default function MediumTitleIntro ({title, textColLeft, textColRight}: MediumTitleIntroProps) {
+export default function MediumTitleIntro ({title, subTitle, textColLeft, textColRight, className}
```

Comment:
WDYM by MediumTitleIntro do we have also a LargeTitleIntro and a SmallTitleIntro ??